src/test/scala/mesosphere/marathon/integration/setup/MarathonCallbackTestSupport.scala

```diff
@@ -81,4 +85,15 @@ trait MarathonCallbackTestSupport extends ExternalMarathonIntegrationTest {
 
     receivedEventsForKinds.groupBy(_.eventType)
   }
+
+  def extractDeploymentIds(app: RestResult[AppDefinition]): Seq[String] = {
```

Comment:
That's not a good place because the topic of this class is actually dealing with callbacks. I also do not like mixins -- especially not for pure methods such as this.